package.json

```diff
@@ -33,9 +33,10 @@
     "audit": "lerna run audit --stream && npm audit --production",
     "bootstrap": "lerna bootstrap",
     "bump": "lerna publish",
+    "commit": "cz",
     "doctoc": "doctoc README.md packages/collector/README.md",
-    "lint": "node misc/is_eol || (npm run typecheck && lerna run lint --stream && eslint misc && eslint dockerfile-examples)",
-    "postinstall": "npm run bootstrap",
+    "lint": "node misc/is_eol || (eslint .*.js *.js && npm run typecheck && lerna run lint --stream && eslint misc && eslint dockerfile-examples)",
```

Comment:
This change (linting additional files and directories) is unrelated to the main intent of this PR.

Reply:
Do you want to move this out of the PR?

Reply:
I would just keep it in this one as it is a really small-ish change unless you feel strongly about having it in a separate PR. The resulting commit history after merging this PR will be the same. (It is a separate commit already.)

Reply:
👍 Works for me